modules/KalturaSupport/resources/mw.KCuePoints.js

```diff
@@ -405,6 +409,54 @@
 			}
 			return foundAttr;
 		},
+		/**
+		 * if have same CP earlier - hide current CuePoint
+		 * @param  allCP - array of CP where try to find same CP
+		 * @param  currentCuePointIndex - position from current CP
+		 *
+		 */
+		removeDuplicatedCuePoints:function (allCP, currentCuePointIndex) {
+			var defaultThreshold = this.getThreshold();
```

Comment:
@MaksymKorniiov  do this @ load time. This never changes

Reply:
We are calling this.getThreshold() with each iteration. If we calculate this @ load time only once it will be more efficient